examples/Peripheral/LED/LED.ino

```diff
@@ -25,6 +25,10 @@ BLELocalDevice BLE(&HCISpiTransport);
 SPIClass SpiHCI(PC12, PC11, PC10);
 HCISpiTransportClass HCISpiTransport(SpiHCI, SPBTLE_RF, PD13, PE6, PA8, 8000000, SPI_MODE0);
 BLELocalDevice BLE(&HCISpiTransport);
+#elif defined(ARDUINO_PNUCLEO_WB55RG)
+/* PNUCLEO_WB55RG */
+HCISharedMemTransportClass HCISharedMemTransport(BLE5_0);
```

Comment:
why BLE5_0 used as parameter ?  what are the other possible values?

Reply:
this "BLE5_0" is chosen to select the RF of the stm32WB55 SIG spec v5.0 to have similar chip identification as with SPI transfer mode

Reply:
The other types in the enum are not based on BLE version but on the selected HW. to be consistent, shouldn't this be something like  STM32WB ?

Reply:
changed to **STM32WB_RF** (as the STM32WB is already defined in the CMSIS stm32wbxx.h

Reply:
How `STM32WB_RF  `is defined ?
Why not simply use the `STM32WBxx` define ?

Reply:
This enum is for defining the BLEChip_s of different boards.
so it's mentionning the RF part of the stm32WB

Reply:
There is one enum type BLE_WB_Chip_t defined for the HCI SharedMem transport of the stm32WB: `STM32WB_RF`
In cas other RF chip are inroduced and to keep the  HCIShardeMemTransportClass declaration  similar to HCISpiTransportClass

Reply:
No more BLE chip type.